test/jdk/java/awt/image/DrawImage/TiledImage.java

```diff
@@ -0,0 +1,201 @@
+import java.awt.Graphics2D;
```

Comment:
The fix itself looks good but the test needs work.
It needs the GPLv2 - like all the other tests, and is missing the jtreg tags which allow the test harness to identify it as a test. and a few orher tags. Again take a look at other tests. Note that since you don't create any windows you will NOT need the @headful tag.